web/src/beta/features/Editor/Tabs/Storytelling/StoryPanel/index.tsx

```diff
@@ -1,18 +1,39 @@
 import { FC } from "react";
 
+import StoryPageIndicator from "@reearth/beta/features/Editor/Tabs/Storytelling/StoryPageIndicator";
 import { styled } from "@reearth/services/theme";
 
 type Props = {};
 
-const StoryPanel: FC<Props> = () => {
-  return <Wrapper>StoryPanel</Wrapper>;
+export const StoryPanel: FC<Props> = () => {
+  return (
+    <Wrapper>
+      <IndicatorWrapper>
+        <StoryPageIndicator
+          currentPage={3}
+          currentPageProgress={33}
+          maxPage={6}
+          onChangePage={page => console.log(page)}
+        />
+      </IndicatorWrapper>
+      <div>StoryPanel</div>
+    </Wrapper>
+  );
 };
 
 export default StoryPanel;
 
 const Wrapper = styled.div`
+  position: relative;
   box-sizing: border-box;
   width: 462px;
   padding: 10px 24px;
   background-color: ${({ theme }) => theme.colors.scrollBar.bg.light};
 `;
+
+const IndicatorWrapper = styled.div`
+  position: absolute;
```

Comment:
If you position it absolutely it'll be outside of the general styles of the Wrapper that the StoryPanel will be a part of, right? 
So that'll mean if I have 10px padding set for each side of the StoryPanel, and then, for example, a Title block at the top, the title will always look too close to the indicator, no?
If the Indicator was hiding unless hovered, your approach here makes sense, but I think it'll always be visible, so not sure we should position absolutely.

Reply:
Thank you, I did it to follow the design but misunderstood it...! Fixed https://github.com/reearth/reearth/pull/543/commits/69ff1a885ccc4eb38d70ff30eaa67b707e60748a